src/routes/settings/Encrypt.tsx

```diff
@@ -66,6 +67,14 @@ export default function Encrypt() {
         }
     };
 
+    const encryptButtonDisabled = createMemo(() => {
+        return (
+            !_encryptPasswordForm.touched ||
```

Comment:
the `_` prefix is so I can keep a variable around unused and keep the linter happy, can remove the underscore

Reply:
didn't know that! tmyk :brain: